### PR TITLE
Filter ajv error messages

### PR DESCRIPTION
**Pull request by adriankast**
Applies filtering duplicate ajv error messages, to reduce unnecessary large output, especially for "oneOf" schema errors.
More details can be found in #25.

Should be merged after #131

### CODE REVIEW

**Review by egekorkan — APPROVED** [review]

**Comment by egekorkan** [comment]
Will merge right after the minor comments are fixed in #131

**Comment by egekorkan** [comment]
Could you push again since #131 is merged? The changes shown are mixed

**Comment by adriankast** [comment]
Created an new PR to separate the changes

**Pull request closed without merging**
